etl/steps/data/garden/biodiversity/2021-01-01/habitat_loss.py

```diff
@@ -0,0 +1,23 @@
+"""Load snapshot and create a garden dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load data from snapshot.
+    #
+    snap = paths.load_snapshot()
+    tb = snap.read(safe_types=False).set_index(["country", "year"])
```

Comment:
fwiw, there is `paths.read_snap_table(...)` function to reduce this to a one-liner